recipes/pyjuliacall/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "pyjuliacall" %}
+{% set pypi_name = "juliacall" %}
+{% set version = "0.9.15" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ pypi_name[0] }}/{{ pypi_name }}/juliacall-{{ version }}.tar.gz
+  sha256: 08163ae1290dda155cdabdccc4035a4cf0c8e361522a5b3f8e6532401b4f29cc
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - setuptools >=42
+    - wheel
+    - pip
+  run:
+    - python >=3.7
+    - pyjuliapkg >=0.1.8,<0.2.dev0
+
+test:
+  imports:
+    - juliacall
+  commands:
+    - pip check
```

Can try importing and using it here

---

Could you elaborate?

---

Something like this:

```suggestion
    - pip check
    - python -c 'from juliacall import Main as jl; assert jl.seval("1 + 1") == 2
```

just to start with. We can always expand later